IndexedDB/bigint_value.htm

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Values</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+
+<script>
+// BigInt and BigInt wrappers are supported in serialization, per
+// https://github.com/whatwg/html/pull/3480
+// This support allows them to be used as IndexedDB values.
+
+let i = 0;
+function value(value, test, name) {
```

Comment:
Can you rename the `test` argument to `test_func` or `predicate`? Since `test` is a testharness function it's confusing to see it re-used as an argument.